phpseclib/Crypt/Hash.php

```diff
@@ -278,20 +278,20 @@ function setHash($hash)
             case 'md2':
                  $this->b = 16;
```

Comment:
Good catch. It seems to be adding in an extra space. This can be seen more clearly in the full code snippet:

```php
        switch ($hash) {
            case 'md2':
                 $this->b = 16;
                 $this->hash = array($this, '_md2');
                break;
```